certification/internal/policy/operator/scc_check.go

```diff
@@ -0,0 +1,120 @@
+package operator
```

Comment:
nit, but none of the other files in this package have '_check' in them. I'd rather that not become a thing.

Reply:
how about scc_info.go and scc_info_test.go ?

Reply:
That's fine for now. Just don't want to introduce _check.